code/controllers/configuration/entries/general.dm

```diff
@@ -645,3 +645,6 @@
 	default = 50
 
 /datum/config_entry/string/morgue_cadaver_override_species
+
+/datum/config_entry/flag/toast_notification_on_init
+	default = FALSE
```

Comment:
i think this its already off by default right

Reply:
huh, I guess you're right.